api/issue.go

```diff
@@ -213,7 +224,15 @@ type IssueFind struct {
 	PipelineID *int
 	// Find issue where principalID is either creator, assignee or subscriber
 	PrincipalID *int
-	StatusList  []IssueStatus
+	// To support pagination, we add into creator, assignee and subscriber.
+	// Only principleID or one of the following three fields can be set.
+	CreatorID    *int
+	AssigneeID   *int
+	SubscriberID *int
+
+	StatusList []IssueStatus
+	// If specified, only find issues whose ID is smaller that MaxID
```

Comment:
Also, shall we make this more generic called tokenID and put another DESC/ASC parameter to control the ordering?

Reply:
According to offline discussions, I just rename the variable to sinceID.